mslib/msui/flighttrack.py

```diff
@@ -616,33 +644,9 @@ def load_from_ftml(self, filename):
         self.load_from_xml_data(xml_content, name)
 
     def load_from_xml_data(self, xml_content, name="Flight track"):
```

Comment:
hmm, we have load_from_xml_data, load_from_ftml,  read_ftml 

is that basically the same?

Reply:
One could probably further tidy that up, but: no
The current three functions are not redundant.